sorc/CMakeLists.txt

```diff
@@ -1,13 +1,30 @@
 include(ExternalProject)
 
+##### CMAKE Arguments depending on exec_bin #########################
+
+if(${exec_bin} STREQUAL exec)
```

Comment:
How about something like this to avoid repetition:
```
list(APPEND COMP_ARGS "-DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}")
if(${exec_bin} STREQUAL exec)
  list(APPEND COMP_ARGS "-DEMC_EXEC_DIR=ON")
endif()
```
My cmake syntax might be wrong, but you get the idea.

Reply:
@gsketefian, It worked. updated.

Reply:
Great, thanks!